diagnostics.md

```diff
@@ -166,6 +166,17 @@ This test depends on the container engine being healthy (see [check_container_en
 Depending on what part of this check failed, there are various fixes and workarounds. Most however will involve a
 restrictive local network, or an unreliable connection.
 
+
+### check_image_corruption
+#### Summary
+This check verifies container image layers stored on the device.  If it fails, at least one image has been corrupted.
+
+#### Triage
+Check that the device has sufficient disk space (see [check_localdisk](#check_localdisk)).  If this does not resolve the issue, it is best to contact support for further assistance.
```

Comment:
We can say that this check also depends on `check_localdisk`, since that'll report out of space issues as well (obviously if a device is already at 100%, we _know_ dispositively this'll fail, but if the device is not quite maxed out yet this still could fail by hitting 100% during the operation).

Reply:
Good point, I'll add that.